arm/Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/readme.md

```diff
@@ -76,6 +77,48 @@ tags: {
 </details>
 <p>
 
+### Parameter Usage: `indexingPolicy`
+
+Tag names and tag values can be provided as needed. A tag can be left without a value.
+
+<details>
+
+<summary>Parameter JSON format</summary>
+
+```json
+"indexingPolicy": {
+    "indexingMode": "consistent",
+    "includedPaths": [
+        {
+            "path": "/*"
+        }
+    ],
+    "excludedPaths": [
+    ]
+}
+```
+
+</details>
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+indexingPolicy: {
+    indexingMode: 'consistent'
+    includedPaths: [
+    {
```

Comment:
Just  very small adjustment; Please indent the includedPaths object one tot he right (like in the JSON example)